addons/sale_project/tests/test_project_profitability.py

```diff
@@ -720,6 +720,13 @@ def test_invoices_without_sale_order_are_accounted_in_profitability(self):
             {
                 'data': [{
                     'id': 'other_invoice_revenues',
+                    'displayLoadMore': False,
+                    'isInvoiceSectionFolded': True,
+                    'sale_items': [{'amount_untaxed_signed': 800.0,
+                                    'id': invoice_1_foreign.id,
+                                    'invoice_date_due': invoice_1_foreign.invoice_date_due,
+                                    'invoice_partner_display_name': 'Georges',
+                                    'name': 'Invoice_1_Foreign',}],
```

Comment:
To be consistent with your changes below:

```suggestion
                    'sale_items': [{
                        'amount_untaxed_signed': 800.0,
                        'id': invoice_1_foreign.id,
                        'invoice_date_due': invoice_1_foreign.invoice_date_due,
                        'invoice_partner_display_name': 'Georges',
                        'name': 'Invoice_1_Foreign',
                    }],
```

By doing that we can easily add another sale_items without altering your line. 🙂